### PR TITLE
VAGOV-1643 Health Care Region News Stories listing page!

### PR DESCRIPTION
**Pull request by aurora-a-k-a-lightning**
## Description
- adding the Health Care Region News Stories listing page!
- re-uses the news story teaser template!
- updated the health care region page to point to `/stories` instead of `/news`!
- updated other links to point to the correct urls!

## Testing done
- Locally with staging data!

## Screenshots
localhost_3001_drupal_pittsburgh-health-care_stories_ (1)

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 1]
```diff
@@ -131,9 +130,9 @@ function paginatePages(page, files, field, layout, ariaLabel, perPage) {
   }
 }
 
-// Return page object with path, breadcrump and title set.
-function createEntityUrl(page, drupalPagePath, title) {
-  const pathSuffix = title.replace(/\s+/g, '-').toLowerCase();
+// Return page object with path, breadcrumb and title set.
+function createEntityUrl(page, drupalPagePath, title, pathSuffix) {
+  pathSuffix = pathSuffix || title.replace(/\s+/g, '-').toLowerCase();
```
Comment (ahay-agile6) [inline src/site/stages/build/drupal/metalsmith-drupal.js]: needed a way to pass in "stories" as the path and "Community stories" as the title!

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 2]
```diff
@@ -224,11 +227,13 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
   );
 
   // Press Release listing page
+  const prEntityUrl = createEntityUrlObj(drupalPagePath);
```
Comment (ahay-agile6) [inline src/site/stages/build/drupal/metalsmith-drupal.js]: fixing breadcrumbs for press releases page

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 3]
```diff
@@ -224,11 +227,13 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
...
   const prObj = Object.assign(
     { allPressReleaseTeasers: page.allPressReleaseTeasers },
     { facilitySidebar: sidebar },
-    { entityUrl: page.entityUrl },
+    { entityUrl: prEntityUrl },
     { title: page.title },
+    { alert: page.alert },
```
Comment (ahay-agile6) [inline src/site/stages/build/drupal/metalsmith-drupal.js]: adding alerts to press releases page

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 4]
```diff
@@ -133,13 +133,16 @@ function paginatePages(page, files, field, layout, ariaLabel, perPage) {
...
-function updateEntityUrlObj(page, drupalPagePath, title) {
-  const pathSuffix = title.replace(/\s+/g, '-').toLowerCase();
+function updateEntityUrlObj(page, drupalPagePath, title, pathSuffix) {
+  pathSuffix = pathSuffix || title.replace(/\s+/g, '-').toLowerCase();
   let generatedPage = Object.assign({}, page);
-  generatedPage.entityUrl.breadcrumb.push({
-    url: { path: drupalPagePath },
-    text: page.title,
-  });
+  generatedPage.entityUrl.breadcrumb = [
```
Comment (ahay-agile6) [inline src/site/stages/build/drupal/metalsmith-drupal.js]: i want to make sure we're not mutating the original page object by creating a brand new array.

File: src/site/stages/build/drupal/metalsmith-drupal.js [hunk 5]
```diff
@@ -238,6 +243,30 @@ function createHealthCareRegionListPages(page, drupalPagePath, files) {
     'press_releases_page.drupal.liquid',
     'press releases',
   );
+
+  // News Story listing page
+  const nsEntityUrl = createEntityUrlObj(drupalPagePath);
+  const nsObj = Object.assign(
```
Comment (jbalboni) [inline src/site/stages/build/drupal/metalsmith-drupal.js]: Do you need to use Object.assign here? I think you could just create a new object. You're not overriding updating a base object, so it doesn't seem like you're gaining anything from this.

**Review by jbalboni — APPROVED** [review]

File: src/site/stages/build/drupal/metalsmith-drupal.js (hunk 5 above)
Comment (ahay-agile6) [reply src/site/stages/build/drupal/metalsmith-drupal.js]: I was thinking the same thing!
I was following the pattern above. If i change it here, should i change it in the above ones as well?

Comment (jbalboni) [reply src/site/stages/build/drupal/metalsmith-drupal.js]: Yeah, I think that would make sense.

**Pull request merged**